Utilities/aliceHLTwrapper/test/testMessageFormat.cxx

```diff
@@ -17,7 +17,7 @@
 #include "Headers/DataHeader.h"
 #include "Headers/HeartbeatFrame.h"
 
-namespace o2::AliceHLT {
+namespace o2 { namespace AliceHLT {
```

Comment:
@ktf: you now that we are testing only a very limited amount of things (and clang-format is not yet tested). A test pass does hence not really mean that everything is fine.